modules/statics/src/base.ts

```diff
@@ -33,6 +33,7 @@ export enum CoinFamily {
   OFC = 'ofc',
   RMG = 'rmg',
   RBTC = 'rbtc',
+  STX = 'stx',
```

Comment:
Can we pull the statics addition out into a new PR please? Will need to merge it individually, then cut a release and update the package.json of account-lib to support the latest statics version (with stacks)

Reply:
I've removed the statics-related commits for now.